python/ray/data/dataset.py

```diff
@@ -2212,6 +2212,7 @@ def sort(
         self,
         key: Union[str, List[str], None] = None,
         descending: Union[bool, List[bool]] = False,
+        boundaries: Optional[list] = None,
```

Comment:
Can you document the arg in the docstring? It'd also be good to specify what the type of the List element should be.

Reply:
Thanks for the tip, I'll provide detailed annotation instructions.